SPECS-EXTENDED/ocaml-xml-light/LICENSE.PTR

```diff
@@ -0,0 +1,25 @@
+License text from the "xml-light/README" file:
```

Comment:
There's no need to copy the text, you can just add `README` to the `%license` macro. What we're looking for here, is to mention that we inferred the LGPLv2.1 license from the fact, that the readme says they are using the OCaml library license and that license is LGPLv2.1. I'd also provide a [link to it](https://ocaml.org/docs/license.html) inside this `LICENSE.PTR` file.

Reply:
Ok got it. Thank you for the explanation. I will update the LICENSE.PTR.